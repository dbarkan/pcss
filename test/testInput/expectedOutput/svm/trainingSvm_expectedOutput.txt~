0	0	N/A	N/A
0.041	0.167	-2.943	0.041
0.116	0.333	-1.898	0.062
0.164	0.5	-1.611	0.068
0.37	0.667	-0.674	0.027
0.623	0.833	0.606	0.021
0.795	1.0	1.608	0.027
1	1	N/A	N/A
