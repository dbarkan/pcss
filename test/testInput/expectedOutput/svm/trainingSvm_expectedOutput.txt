0	0	N/A	N/A
0.11	0.167	-2.198	0.014
0.158	0.333	-1.677	0.048
0.212	0.5	-1.292	0.021
0.267	0.667	-1.064	0.034
0.541	0.833	0.125	0.062
0.877	1.0	2.096	0.041
1	1	N/A	N/A
Critical True Positive Rate: 0.833
Critical False Positive Rate: 0.541
Score at critical point: 0.125
